source/assets/public/footer/index.css

```diff
@@ -39,6 +30,11 @@
   margin-bottom: 2rem;
 }
 
-.breakpoint-phone .footer-nav__list-item a {
-  margin-bottom: 1rem;
-}
+@media (max-width:639px) {
```

Comment:
Can we use variables for these breakpoints?

Reply:
Can do!